### PR TITLE
Backend/aggregator group

### PR DESCRIPTION
**Pull request by TWEApol**
This PR adds the `fetch_aggregator_daterange` method to `AggregatorGroup`.
This method requires you to specify a `start_datetime` and `end_datetime`, where the range of dates would be [`start_datetime`, `end_datetime`), such that the returned Aggregator object contains the aggregated values of all aggregators/endpoint-calls within that datetime range.

The method will be precise up to the minute in the sense that it disregards all data from seconds and downwards in terms of precision. (e.g. `datetime(year=1, month=2, day=3, hour=4, minute=5, second=6, etc.)` is regarded to be the same as `datetime(year=1, month=2, day=3, hour=4, minute=5, second=0, etc.)` )

### CODE REVIEW

File: pydash/pydash_app/dashboard/aggregator/aggregator_group.py [hunk 1]
```diff
@@ -177,16 +212,187 @@ def fetch_aggregator(self, partition_field_names):
...
+        for key, value in date_chunks.items():
+            filters_cpy = copy(filters)
+            for datetime in value:
+                filters_cpy[key] = datetime.strftime(datetime_formats[key])
+                aggregator += self.fetch_aggregator(filters_cpy)
+
+        return aggregator
+
+
+def chop_date_range_into_chunks(datetime_begin, datetime_end):
```
Comment (InputUsername) [inline pydash/pydash_app/dashboard/aggregator/aggregator_group.py]: I'm not a big fan of the formatting of this function; a lot of whitespace that makes it a bit hard to read everything. Maybe decrease the indentation level a bit?

Comment (InputUsername) [reply pydash/pydash_app/dashboard/aggregator/aggregator_group.py]: This is just aesthetic though, the functionality looks good :+1:

Comment (TWEApol) [reply pydash/pydash_app/dashboard/aggregator/aggregator_group.py]: The reason why I have so much whitespace is that I tried to keep everything structured, while trying not to write very long lines of code.

Comment (InputUsername) [reply pydash/pydash_app/dashboard/aggregator/aggregator_group.py]: In this case I'd say writing a couple lines more would be preferable to having lines that are 99% whitespace :wink:

**Pull request merged**